src/retype/__init__.py

```diff
@@ -1143,6 +1145,16 @@ def copy_type_comment_to_annotation(arg):
     arg.annotation = ann
 
 
+def normalize_strings_to_repr(node):
+    """Normalize string leaf nodes to a repr since that is what we generate."""
+    if node.type == token.STRING:
+        node.value = repr(ast3.literal_eval(node.value))
+    elif isinstance(node, Node):
+        node.children = [normalize_strings_to_repr(i) for i in node.children]
+
+    return node
+
+
 def maybe_replace_any_if_equal(name, expected, actual, flags):
```

Comment:
I'm a little unsure of this function... if the strings are equal shouldn't the original be used rather than the one we generated? I believe the return should probably be `actual` not `expected` (and I'm not sure why a fallback is given, but it could be given in reverse order).

I'm also a little confused about the `Any` normalization. From the comment it seems like people should be able to keep their original `Any`, but if you don't use `--replace-any` then it will error out instead of leaving things as is, but if you use `--replace-any` it doesn't check for nested `Any`s and was very naively stripping quotes. I'm guessing it _probably_ was OK, but I updated this to look at the node type in order to strip the quotes and used the normalized string so that it will always be a single single quote (I'm not sure if `"""` or `'''` are allowed for `Leaf(STRING)` values).

Reply:
Hmm, I can't decide on this. This part was mostly written by Lukasz so don't have a good answer here.

Reply:
I'm not in a rush on this, and I'm _thinking_ we should probably be recursively checking for `Any` in case it's wrapped in something like `Optional[Any]` or `Dict[str, Any]` etc, but I'm not sure if `--replace-any` is targeting *strictly* `Any` that is not contained in anything else.

This code also seems like it's allowing a string form of Any to be normalized into a non string form of Any so checking nodes directly would not be sufficient (and why it's `str(node)` first) since `Leaf(STRING, "'Any'") != Leaf(NAME, "Any")` (and other Any forms that are captured) so if it was recursively allowing Any it would not be as trivial as looking for `Leaf(String)` and normalizing `Any`s to a single form "like" what I'm doing to normalize to the `repr` form.